File: class20/python/AP2002-Exemplo02.py
# TrianguloFloyd — Aula 20
# Triângulo de Floyd (números naturais em triângulo)

n = int(input("Número de linhas: "))
num = 1
for i in range(1, n + 1):
    for j in range(i):
        print(f"{num:3}", end="")
        num += 1
    print()

File: class20/cpp/AP2002-Exemplo02.cpp
#include <iostream>
#include <iomanip>
int main() {
    int n, num = 1;
    std::cout << "Número de linhas: "; std::cin >> n;
    for (int i = 1; i <= n; i++) {
        for (int j = 0; j < i; j++)
            std::cout << std::setw(3) << num++;
        std::cout << std::endl;
    }
    return 0;
}